### DISCUSSION TITLE
Pivot Grid - show empty rows function

### DISCUSSION THREAD

**Discussion by MarcBasten** [TabularEditor/TabularEditor3#1226] [Other Topics]
Hi, i am trying to filter out empty rows and columns of my pivot grid. However clicking the button "show empty values on rows" etc. does not work for me. Any idea how to filter out all the empty rows and columns?

**Comment by otykier** [comment]
When I tested this just now using 3.13, it seems to work as expected. Pivot Grid on the right has "Show empty values on rows" enabled:

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/a8abc294-a819-4752-830b-70e6c72e0904)

Could you share some more details on how your Pivot Grid is set up, and possibly the DAX expression making up the measure you're using?

**Reply by MarcBasten** [reply]
Hi Daniel, thanks for the fast reply! You were right. I was checking the base measures, which change empty values into '0' (as mgmt does not want to see empty fields). I guess if filtering zeros is not possible i have to add an extra measure for working with the pivot grid. 

Again, thanks for the fast reply :)
